MekHQ/src/mekhq/campaign/unit/Unit.java

```diff
@@ -4711,74 +4717,87 @@ public boolean isPresent() {
     }
 
     public int getMaintenanceTime() {
+        int retVal = 0;
```

Comment:
Can we please call this time or maintenanceTime instead of retVal?

Reply:
Really?

Reply:
Yes. I should be able to tell what a variable is from a glance. retVal tells me nothing as to what the value is, just that it is the one returned (and is a misnomer in this case, as it returns while being multiplied)

Reply:
Sounds like a personal preference. Pretty sure it's not going to make a difference in this method, since there are no other local variables anyway.

Reply:
It's best practices for variable names in general, not a personal preference.

Reply:
Best practices such as these are in the eye of the beholder, and you can easily argue this either way.

Reply:
In that case... please tell me how retVal is better than time or maintenanceTime for a variable name.

Reply:
"This is the value you will see coming back at the end of this method." has roughly same comprehension value as "This is the maintenance time." Therefore, it's not worth changing. If you want to name variables in code you write using a different naming convention, that's your call, but I can assure you that, unless it's actively misleading or unprofessional, nobody will give you a hard time about it.

Reply:
It's a misnomer, as that's not the value you return.

As for naming conventions... I'd highly disagree. Making it understandable is key for future proofing. I'll agree to disagree though, to get this in.